src/test/java/com/williamfiset/algorithms/datastructures/queue/QueueTest.java

```diff
@@ -1,71 +1,87 @@
 package com.williamfiset.algorithms.datastructures.queue;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.*;
 
+import java.util.ArrayList;
+import java.util.List;
 import org.junit.Before;
 import org.junit.Test;
 
 public class QueueTest {
 
-  Queue<Integer> queue;
+  private List<Queue<Integer>> queues = new ArrayList<>();
 
   @Before
   public void setup() {
-    queue = new Queue<Integer>();
+    queues.add(new ArrayQueue<Integer>(2));
+    queues.add(new LinkedQueue<Integer>());
```

Comment:
Let's add the IntQueue? and for the Stack tests the IntStack?

Reply:
done